src/include/sof/drivers/sai.h

```diff
@@ -251,8 +251,14 @@
  * configured for an internal bit clock.
  * The division value is (DIV + 1) * 2.
  */
+#ifndef CONFIG_IMX8ULP
 #define SAI_CLOCK_DIV		0x7
 #define SAI_TDM_SLOTS		2
```

Comment:
put the condition the other way around #ifdef CONFIG_IMX8ULP. I'm fine with this for now, but in the future we should make